CHANGES.rst

```diff
@@ -29,6 +29,47 @@ vizier
 - Change the type of raised error when the catalog is not found in ``Vizier.get_catalog_metadata``
   from ``IndexError`` to ``EmptyResponseError`` [#2980]
 
+simbad
+^^^^^^
+
+- The ``ROW_LIMIT`` value to have the maximum number of rows is now -1 instead of 0. This
+  allows to retrieve an empty table with the output's meta-data [#2954]
+
+- ``ROW_LIMIT`` can now be set at instantiation (ex: ``simbad = Simbad(ROW_LIMIT=10))``) [#2954]
+
+- ``list_votable_fields`` now return an astropy Table with added fields information instead
+  of a list of strings [#2954]
+
+- ``list_votable_fields`` is now queried directly from SIMBAD instead of reading a file
+  in astroquery. This prevents it from being outdated [#2954]
+
+- ``get_votable_fields`` now prints the table name and column name instead of just the
+  column name [#2954]
+
+- all query methods now accept ``get_adql`` boolean argument that returns the ADQL string
+  instead of sending the request to SIMBAD. The ``verbose`` and ``get_query_payload``
+  are removed from all methods [#2954]
```

Comment:
@bsipocz big picture question here - might it be better to keep `get_query_payload` as a duplicate of `get_adql` here?  I liked that this value was consistent across all modules.  It's primarily for debugging, so I think we're the primary customers, but I'd like your take on this.

Reply:
I'm ok with keeping both too, it's just that `get_query_payload` will return the ADQL string instead of the payload. Maybe a true request payload is something to add in PyVO's TAP module ?

Reply:
Right, if it's the payload, it's going to be something like `{'adql': <text of adql query>}`.  

It's OK if it's just the ADQL, though.  The payloads tend to be different for every module, though I think they are all `dict`s.

Reply:
I read a bit more the TAP module in pyVO. It's actually possible to get the payload (!)
So it's done and we now have: 

```python
from astroquery.simbad import Simbad
Simbad.query_tap("select top 1 * from basic", get_query_payload=True)
```

```
{'REQUEST': 'doQuery', 'LANG': 'ADQL', 'MAXREC': 10000, 'QUERY': 'select top 1 * from basic'}
```

available for every query method in the module :)